i18n.php

```diff
@@ -921,7 +921,7 @@
     'but' => 'Nhưng',
     'examples' => 'Dữ liệu',
     'feature' => 'Tính năng',
-    'given' => 'Biết|Cho',
+    'given' => '*|Biết|Cho',
```

Comment:
this file must not be edited by hand. We have a script generating it (which needs to be updated btw, which is on my TODO list). Manual updates on this file will be rejected